scripts/update_configs.py

```diff
@@ -86,23 +100,94 @@ def _apply_changes_based_on_config(
         writable_local_file.write('\n'.join(local_lines) + '\n')
 
 
-def _update_configs():
-    """Updates the 'feconf.py' and 'constants.ts' files after doing the
+def check_updates_to_terms_of_service():
+    """Checks if updates are made to terms of service and updates
+    REGISTRATION_PAGE_LAST_UPDATED_UTC in feconf.py if there are updates.
+    """
+    personal_access_token = common.get_personal_access_token()
+    g = github.Github(personal_access_token)
+    repo = g.get_organization('oppia').get_repo('oppia')
+
+    common.open_new_tab_in_browser_if_possible(
+        'https://github.com/oppia/oppia/commits/develop/core/'
+        'templates/dev/head/pages/terms-page/terms-page.mainpage.html')
+    python_utils.PRINT(
+        'Are the terms of service changed? Check commits/changes made '
+        'to the file: terms-page.mainpage.html. Enter y/ye/yes if they '
+        'are changed else enter n/no.')
+    is_terms_of_service_changed = python_utils.INPUT().lower()
```

Comment:
terms_of_service_are_changed (it's a boolean, not a function)

Reply:
Done.